charts/alluxio/templates/fuse/daemonset.yaml

```diff
@@ -56,38 +56,6 @@ spec:
       {{- end }}
       securityContext:
         fsGroup: {{ .Values.fuse.fsGroup }}
```

Comment:
Please update chart version and add changelog. thanks.

Reply:
Updated